ui/selectors/nft.ts

```diff
@@ -1,21 +1,30 @@
-import { NftContract } from '@metamask/assets-controllers';
+import { Nft, NftContract } from '@metamask/assets-controllers';
 import { createSelector } from 'reselect';
 import { getMemoizedCurrentChainId } from './selectors';
 
-type NftState = {
+export type NftState = {
   metamask: {
     allNftContracts: {
       [account: string]: {
         [chainId: string]: NftContract[];
       };
     };
+    allNfts: {
+      [account: string]: {
+        [chainId: string]: Nft[];
+      };
+    };
   };
 };
 
 function getNftContractsByChainByAccount(state: NftState) {
   return state.metamask.allNftContracts ?? {};
 }
 
+function getNftsByChainByAccount(state: NftState) {
```

Comment:
Thoughts on moving both of these new selectors into `ui/ducks/metamask`? It may end up getting moved there soon anyway as part of the circular dependency work

Reply:
I always try to avoid adding to their monolithic selector files. I appreciate our temporary solution is to merge them again for the circular dependency work, but my thinking was it's better to record where this conceptually belongs so we can come back to a clean isolated selector in future.

The only dependency so far is `getMemoizedCurrentChainId` which should be removed for multi-chain anyway.